### PR TITLE
Tutorial and HowTo clean up

### PR DESCRIPTION
**Pull request by hhariri**

### CODE REVIEW

File: pages/docs/tutorials/native/using-command-line-compiler.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+<!--- To become a How-To. Need to change type to new "HowTo" --->
+
+
+## Obtaining the Compiler
+
+The Kotlin/Native compiler is available for macOS, Linux, and Windows. It supports
+different targets including iOS (arm32, arm64, simulator x86_64), Windows (mingw32 and x86_64),
+Linux (x86_64, arm64, MIPS), macOS (x86_64), Raspberry PI, SMT32, WASM. 
+
+For the full list of targets please see the [Kotlin/Native overview](/docs/reference/native-overview.html).
```
Comment (p7nov) [inline pages/docs/tutorials/native/using-command-line-compiler.md]: I would add this to the previous paragraph and start a new one after.

File: pages/docs/tutorials/native/using-command-line-compiler.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+description: "A look at how to compileKotlin/Native applications using the command line compiler"
+authors: 
+  - Hadi Hariri
+date: 2020-01-15
+---
+
+<!--- To become a How-To. Need to change type to new "HowTo" --->
+
+
+## Obtaining the Compiler
```
Comment (p7nov) [inline pages/docs/tutorials/native/using-command-line-compiler.md]: Couldn't find anything about obtaining the compiler in this section. Mabye add a couple of sentences explaining what is it (like: a CLU util, we ship it in Kotlin distrib with each release, etc.) and where to get it.

File: pages/docs/tutorials/native/using-command-line-compiler.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+## Compiling the code from the console 
+
+To compile the application use the [downloaded](https://github.com/JetBrains/kotlin/releases)
+compiler to execute the following command:
+
+```bash
+kotlinc-native hello.kt -o hello
+```
+
+This should generate a `hello.kexe` (Linux and macOS) or `hello.exe` (Windows) binary file
```
Comment (p7nov) [inline pages/docs/tutorials/native/using-command-line-compiler.md]: Maybe add a gif how you execute the file?
Also, please add a period at the end of the sentence.

Comment (p7nov) [reply pages/docs/tutorials/native/using-command-line-compiler.md]: One more suggestion: a link to the compiler reference https://kotlinlang.org/docs/reference/compiler-reference.html

File: pages/docs/tutorials/native/using-command-line-compiler.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+While compilation from the console seems to be easy and clear, it
+does not scale well for larger projects with hundreds of files and libraries. For real-world projects it is recommended
+to use a [build system](using-gradle.html) and [IDE](using-intellij-idea.html)
```
Comment (p7nov) [inline pages/docs/tutorials/native/using-command-line-compiler.md]:
```suggestion
to use a [build system](using-gradle.html) and [IDE](using-intellij-idea.html).
```

File: pages/docs/tutorials/native/using-command-line-compiler.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
+---
+type: tutorial
+layout: tutorial
+title:  "Hello Kotlin/Native using Command Line Compiler"
+description: "A look at how to compileKotlin/Native applications using the command line compiler"
```
Comment (p7nov) [inline pages/docs/tutorials/native/using-command-line-compiler.md]:
```suggestion
description: "A look at how to compileKotlin/Native applications using the command line compiler."
```

File: pages/docs/tutorials/native/using-gradle.md [hunk 6]
```diff
@@ -0,0 +1,93 @@
+---
+type: tutorial
+layout: tutorial
+title:  "Hello Kotlin/Native using Gradle"
+description: "A look at how to compile Kotlin/Native applications using Gradle"
```
Comment (p7nov) [inline pages/docs/tutorials/native/using-gradle.md]:
```suggestion
description: "A look at how to compile Kotlin/Native applications using Gradle."
```

File: pages/docs/tutorials/native/using-gradle.md [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+
+This should create a folder `build/native/bin` with two subfolders `debugExecutable` and `releaseExecutable` with the corresponding binary.
+By default, the binary is named the same as the project folder. 
+
+
+## Opening the project in an IDE
+
+Any IDE that supports Gradle should allow for opening the project in the IDE. In the case of [IntelliJ IDEA](https://www.jetbrains.com/idea), just open the project folder and it will automatically
+detect it as Kotlin/Native project. 
+
```
Comment (p7nov) [inline pages/docs/tutorials/native/using-gradle.md]: Need a `Learn more` on `Next step` section with a link to https://kotlinlang.org/docs/reference/building-mpp-with-gradle.html

File: pages/docs/tutorials/native/using-intellij-idea.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
+---
+type: tutorial
+layout: tutorial
+title:  "Hello Kotlin/Native using IntelliJ IDEA"
+description: "A look at how to a Kotlin/Native application using IntelliJ IDEA"
```
Comment (p7nov) [inline pages/docs/tutorials/native/using-intellij-idea.md]:
```suggestion
description: "A look at how to create a Kotlin/Native application using IntelliJ IDEA."
```

File: pages/docs/tutorials/native/using-intellij-idea.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+<!--- To become a How-To. Need to change type to new "HowTo" --->
+
+
+## Creating a new Kotlin/Native project in IntelliJ IDEA
+
+This following is applicable to both [IntelliJ IDEA Community Edition as well as the Ultimate Edition](https://www.jetbrains.com/idea).
+
+
+From the **File** menu in IntelliJ IDEA or **Welcome screen**, select to create a new Project and in the first step of the Wizard
+select Kotlin on the left hand column and Kotlin/Native on the right hand.
```
Comment (p7nov) [inline pages/docs/tutorials/native/using-intellij-idea.md]:
```suggestion
select **Kotlin** on the left hand column and **Native | Gradle** on the right hand.
```

File: pages/docs/tutorials/native/using-intellij-idea.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+
+![Wizard Step One]({{ url_for('tutorial_img', filename='native/using-intellij-idea/wizard.png')}})
+
+Click Next in the dialog and in the next step make sure `Automatically import this project on changes in build script` is checked. This is useful when starting
```
Comment (p7nov) [inline pages/docs/tutorials/native/using-intellij-idea.md]:
```suggestion
Click **Next** in the dialog and in the next step make sure **Automatically import this project on changes in build script** is checked. This is useful when starting
```

File: pages/docs/tutorials/native/using-intellij-idea.md [hunk 11]
```diff
@@ -0,0 +1,43 @@
...
+out to make sure any immediate changes to the build script are imported automatically. 
+
+![Wizard Step Two]({{ url_for('tutorial_img', filename='native/using-intellij-idea/wizard-2.png')}})
+
+On clicking Next, enter the path and name for the project
```
Comment (p7nov) [inline pages/docs/tutorials/native/using-intellij-idea.md]:
```suggestion
On clicking **Next**, enter the path and name for the project.
```

File: pages/docs/tutorials/native/using-intellij-idea.md [hunk 12]
```diff
@@ -0,0 +1,43 @@
...
+system the project was created on (Windows, Linux, macOS).
+
+![Project]({{ url_for('tutorial_img', filename='native/using-intellij-idea/IDE-1.png')}})
+
+To run the project, simply invoke the [Run command in the IDE](https://www.jetbrains.com/help/idea/running-applications.html), by using the corresponding shortcuts or from the Run menu
+
+![Run]({{ url_for('tutorial_img', filename='native/using-intellij-idea/IDE-2.png')}})
+
+The sample project can serve as the basis for any new project for Kotlin/Native.
+
```
Comment (p7nov) [inline pages/docs/tutorials/native/using-intellij-idea.md]: Some useful links? Maybe these:
https://kotlinlang.org/docs/reference/native/gradle_plugin.html
https://kotlinlang.org/docs/reference/building-mpp-with-gradle.html

File: data/_nav.yml [hunk 13]
```diff
@@ -384,8 +384,12 @@ tutorials:
       content:
         - url: /docs/tutorials/native/intro-to-kotlin-native.html
           title: Introduction to Kotlin/Native
```
Comment (p7nov) [inline data/_nav.yml]: The Hands-on under this link seems more advanced than the next three topics. Should we rename the link to reflect content (HTTP client) and move in lower in the list?

File: pages/docs/tutorials/native/using-command-line-compiler.md (hunk 2 above)
Comment (hhariri) [reply pages/docs/tutorials/native/using-command-line-compiler.md]: Added.

File: pages/docs/tutorials/native/using-command-line-compiler.md (hunk 4 above)
Comment (hhariri) [reply pages/docs/tutorials/native/using-command-line-compiler.md]: Fixed

File: pages/docs/tutorials/native/using-intellij-idea.md (hunk 9 above)
Comment (hhariri) [reply pages/docs/tutorials/native/using-intellij-idea.md]: Will fix.

File: pages/docs/tutorials/native/using-intellij-idea.md (hunk 10 above)
Comment (hhariri) [reply pages/docs/tutorials/native/using-intellij-idea.md]: Will fix.

File: pages/docs/tutorials/native/using-intellij-idea.md (hunk 11 above)
Comment (hhariri) [reply pages/docs/tutorials/native/using-intellij-idea.md]: Will fix.

File: data/_nav.yml (hunk 13 above)
Comment (hhariri) [reply data/_nav.yml]: Yes. Agree. Will do.

**Pull request merged**